pkgs/development/compilers/c3c/default.nix

```
@@ -3,24 +3,33 @@
 , fetchFromGitHub
 , cmake
 , python3
+, curl
+, libxml2
+, libffi
+, xar
 }:
 
 llvmPackages.stdenv.mkDerivation rec {
   pname = "c3c";
-  version = "unstable-2021-07-30";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "c3lang";
     repo = pname;
-    rev = "2246b641b16e581aec9059c8358858e10a548d94";
-    sha256 = "VdMKdQsedDQCnsmTxO4HnBj5GH/EThspnotvrAscSqE=";
+    rev = "3255183ee49cb903be1c66717d90f35cbcae7f82";
```

In this case we should be able to point to the tag directly.

```suggestion
    rev = "refs/tags/v${version}";
```